lib/detect-testing-library-utils.ts

```diff
@@ -176,6 +187,58 @@ export function detectTestingLibraryUtils<
       return ASYNC_UTILS.includes(node.name);
     };
 
+    /**
+     * Determines whether a given node is fireEvent method or not
+     */
+    const isFireEventMethod: DetectionHelpers['isFireEventMethod'] = (node) => {
```

Comment:
Extracted from `prefer-user-event`! I tweaked it a little bit to improve some edge cases.